consensus/src/epoch_manager.rs

```diff
@@ -845,13 +846,13 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
         );
         let (round_manager_tx, round_manager_rx) = aptos_channel::new(
             QueueStyle::LIFO,
-            10,
+            100,
```

Comment:
100 is too many messages, don't bump things arbitrarily. this is per peer per message type. but we should fix the queue order

Reply:
Done.